tests/meson.build

```diff
@@ -77,7 +81,8 @@ foreach test_name : pistache_test_files
 				gmock_dep,
 				curl_dep,
 				cpp_httplib_dep,
-				brotli_dep
+				brotli_dep,
+        zstd_dep
```

Comment:
Remember that `libzstd-dev` also needs to be added to `Build-Depends` in `debian/control`. That's in a separate branch and so you'll need a separate PR for that.

Reply:
Hi @EdgarModesto23 -

I see a number of the github workflows (https://github.com/pistacheio/pistache/actions) are failing, e.g. with:
`tests/meson.build:71:1: ERROR: Unknown variable "zstd_dep".
`
Some of that may be related to the need for an update to debian/control. Certainly the way brotli is dealt with should be a good model.

The workflows _linuxflibev_, _linux_ and _macOS libevent_ should all pass (occasionally the debian:testing tests under linux workflow will fail if debian:testing is in a funny state, but in fact it seems OK at present). The _conventional-commits_ and _abidiff_ are less important (just make sure you bump version number if _abidiff_ changes).